src/sinks/clickhouse/native/convert.rs

```diff
@@ -18,15 +18,15 @@ pub(super) fn build_block(
 ) -> crate::Result<Block> {
     let mut b = Block::new();
     for ev in &events {
-        b.push(get_row_from_events(&schema, &ev.clone().into_log())?)
+        b.push(get_row_from_events(&schema, ev.clone().into_log())?)
```

Comment:
Should be able to remove the clone:

```suggestion
        b.push(get_row_from_events(&schema, ev.into_log())?)
```

Reply:
Actually, no. I have tried previously to di it, but got the following error:
```
error[E0507]: cannot move out of `*ev` which is behind a shared reference
   --> src/sinks/clickhouse/native/convert.rs:21:45
    |
21  |         b.push(get_row_from_events(&schema, ev.into_log())?)
    |                                             ^^^----------
    |                                             |  |
    |                                             |  `*ev` moved due to this method call
    |                                             move occurs because `*ev` has type `vector_core::event::Event`, which does not implement the `Copy` trait
    |
note: this function takes ownership of the receiver `self`, which moves `*ev`
   --> /Users/zamazan4ik/open_source/vector/lib/vector-core/src/event/mod.rs:123:21
    |
123 |     pub fn into_log(self) -> LogEvent {
    |                     ^^^^

```

Reply:
ah, I thought you had removed the ampersand from the line above. That is needed as well:

```Rust
for ev in events {
    b.push(get_row_from_events(&schema, ev.into_log())?)
}

Reply:
Ahh, I see, sorry - my bad. I've overlooked that we can easily take ownership over `event` during the iteration since the `events` are not used anymore later in this function. Fixed.